modules/samples/oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/ApplicationConfig.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.sample.identity.oauth2;
+
+import org.apache.commons.lang.StringUtils;
+
+import java.util.Properties;
+import java.util.logging.Logger;
+
+/**
+ * Sample OAuth2 application config that read from file based.
```

Comment:
Since this class doesn't do the actual file reading, shall we change the class comment?

Reply:
Thanks Rushmin.. Done that.